storage/innobase/fil/fil0fil.cc

```diff
@@ -2803,22 +2803,23 @@ dberr_t Fil_shard::get_file_size(fil_node_t *file, bool read_only_mode) {
   }
 
   if (file->size == 0) {
+#ifndef XTRABACKUP
```

Comment:
why was this moved a few lines above?

Reply:
it throwing compilation error extent_size being unused